### ISSUE TITLE
Scale tests leave dirty mme_nas_state

### ISSUE THREAD

**Issue by ulaskozat** [magma/magma#7036]
In scaling tests we end up in duplicate attach requests and ignoring legitimate users. Inspecting redis state shows dirty guti map although there are no sessions in the system:

```
magma@1vagw2:~$ state_cli.py parse "mme_nas_state" | grep guti_ue_id_htbl | wc -l
1429
```

**Comment by rsarwad** [comment]
Hi @ulaskozat  can we get mme.log and wireshark capture

**Comment by ulaskozat** [comment]
@rsarwad I do not have the pcaps but the mme log and the state info.

[log.zip](https://github.com/magma/magma/files/6524434/log.zip)

**Comment by rsarwad** [comment]
Hi @ulaskozat  I am not able to view the mme.log attached within log.zip.  It's appearing as below 
![image](https://user-images.githubusercontent.com/30685618/119281247-9e0e2780-bc52-11eb-8d0f-9a5fe9f14857.png)

**Comment by ulaskozat** [comment]
@rsarwad most likely I copied mme binary rather than the mme log message. I will rerun the test and capture pcaps as well.

**Comment by ulaskozat** [comment]
@rsarwad attached the new logs and pcaps. I also included leftover states. There are some on MME and some on SPGW. There are a good number of session rejects due to IP address allocation failures I think.

[log_300UE_10UEpsec.zip](https://github.com/magma/magma/files/6529808/log_300UE_10UEpsec.zip)

[sim_300UE_rate10.pcapng.zip](https://github.com/magma/magma/files/6529812/sim_300UE_rate10.pcapng.zip)

**Comment by ardzoht** [comment]
To follow up on this, I ran a test with 500 UEs / 5 UE per sec attach for 30 iterations (which took around 10 hours), I monitored over time the changes on state, SPGW and MME UE state maintained the same number of entries as the `get_subscriber_table` command. Also the `nb_ue_attached` on `mme_nas_state` followed the same pattern.

These are the results at the end of the runs:

```
magma@phy-u6:~$ state_cli.py keys "*:SPGW" | wc -l
0
magma@phy-u6:~$ state_cli.py keys "*:MME" | wc -l
0
magma@phy-u6:~$ state_cli.py keys "*:S1AP" | wc -l
0
magma@phy-u6:~$ state_cli.py parse s1ap_state | grep mmeid2associd | wc -l
0
magma@phy-u6:~$ state_cli.py parse "mme_nas_state" | grep guti_ue_id_htbl | wc -l
7
magma@phy-u6:~$ state_cli.py parse s1ap_state | grep mmeid2associd | wc -l
0
magma@phy-u6:~$ mobility_cli.py get_subscriber_table | wc -l
1 => (header row)
magma@phy-u6:~$ state_cli.py parse mme_nas_state | grep nb_ue_attached
0
magma@phy-u6:~$ state_cli.py parse mme_nas_state | grep tun11_ue_id_htbl | wc -l
6
```

**Comment by ardzoht** [comment]
Closing this since fix for stale GUTI and tun11 hashtable entries was merged on #7721
